src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -621,12 +622,12 @@ public void onTraversalStart() {
 
         if (doIncrementalImport) {
             logger.info("Callset Map JSON file will be re-written to " + callsetMapJSONFile);
-            logger.info("Incrementally importing to array - " + workspaceDir + "/" + GenomicsDBConstants.DEFAULT_ARRAY_NAME);
+            logger.info("Incrementally importing to workspace - " + workspaceDir);
         } else {
             logger.info("Vid Map JSON file will be written to " + vidMapJSONFile);
             logger.info("Callset Map JSON file will be written to " + callsetMapJSONFile);
             logger.info("Complete VCF Header will be written to " + vcfHeaderFile);
-            logger.info("Importing to array - " + workspaceDir + "/" + GenomicsDBConstants.DEFAULT_ARRAY_NAME);
```

Comment:
Should the constant `GenomicsDBConstants.DEFAULT_ARRAY_NAME` be itself removed as well?

Reply:
I looked at removing this constant. But, this is needed to support querying older GenomicsDB arrays.